client/asset/interface.go

```diff
@@ -358,10 +375,12 @@ type Recoverer interface {
 	Move(backupdir string) error
 }
 
-// Sender is a wallet that can send funds to an address, as opposed to
-// withdrawing a certain amount from the source wallet/account.
-type Sender interface {
-	Send(address string, value, feeSuggestion uint64) (Coin, error)
+// Withdrawer is a wallet that can withdraw a certain amount from the
+// source wallet/account.
+type Withdrawer interface {
+	// Withdraw withdraws funds to the specified address. Fees are subtracted
+	// from the value.
+	Withdraw(address string, value, feeSuggestion uint64) (Coin, error)
```

Comment:
Why back to `feeSuggestion` on these?  These methods used to do fee estimation internally and use the suggestion as a fallback, but they were changed to take the `feeRate` that they should just use, if not zero: https://github.com/decred/dcrdex/commit/e41ecdfe57045cd3bf6b22aa7a6054740f0dca05
I see the eth backend still said `feeSuggestion` on the impl, but I think that is just a minor deviation in the naming.

That was a somewhat contentious change, but that's what we did.  https://github.com/decred/dcrdex/pull/1419

Reply:
Thanks for referencing #1419 and `e41ecdf` didn't know `feeSuggestion` had special meaning.